amr-wind/wind_energy/actuator/wing/wing_ops.H

```diff
@@ -118,14 +118,25 @@ struct ComputeForceOp<
         amrex::Real total_lift = 0.0;
         amrex::Real total_drag = 0.0;
         for (int ip = 0; ip < npts; ++ip) {
-            const auto& tmat = grid.orientation[ip];
-            // Effective velocity at the wing control point in local frame
-            auto wvel = tmat & grid.vel[ip];
-            // Set spanwise component to zero to get a pure 2D velocity
-            wvel.y() = 0.0;
 
-            const auto vmag = vs::mag(wvel);
-            const auto aoa = std::atan2(wvel.z(), wvel.x());
+            // Build the local reference frame
+            vs::Vector wspan = wdata.end - wdata.start;
+            wspan = wspan.unit();
+
+            // Use the global coord to orient the blade
+            // This works only for inflow in the x direction
```

Comment:
If it will only work for a single direction then we need a check in place to ensure that is the only way it can be run.

Reply:
Thanks for noticing that @psakievich !
This was an old comment and the new formulation works for more directions.
I still need to put some thought to make sure that the wing can work in any direction without a problem. 
Will fix this tomorrow.